src/test/java/org/apache/commons/codec/binary/Base64Test.java

```diff
@@ -1309,6 +1309,11 @@ public void testUrlSafe() {
 
     }
 
+    public void testCodec263(){
+      Base64.decodeBase64("publishMessage");
```

Comment:
I would make it more obvious IMO to refactor the magic strings into a local variable (or a constant).